bigquery/noxfile.py

```diff
@@ -40,6 +40,13 @@ def default(session):
 
     session.install("-e", os.path.join("..", "test_utils"))
     dev_install = ".[all]"
+    coverage_fail_under = "--cov-fail-under=97"
+
+    # There is no pyarrow or fastparquet wheel for Python 3.8.
+    if session.python == "3.8":
+        dev_install = ".[pandas,tqdm]"
+        coverage_fail_under = "--cov-fail-under=93"
```

Comment:
Maybe add a comment on why a lower threshold coverage here? And a TODO reminder to remove it once the missing wheels become available?